tests/test_connection_string.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+
+from tap_mongodb import get_connection_string
+
+
+class TestConnectionString(unittest.TestCase):
```

Comment:
Can you fix the indentation in this file? it's currently using 2 spaces rather than 4

Reply:
Sorry about that @Samira-El. It's fixed now.

Reply:
No worries, thanks!